mandatory/2-logic-error.js

```diff
@@ -1,16 +1,15 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
-function trimWord(word) {
-  return wordtrim();
+function trimWord(  word  ) {
+  return word.trim(word);
```

Comment:
function trimWord(word) {
  return word.trim();
}

Reply:
Have a look at the documentation for trim here - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim.
You don't actually need to pass any parameters to this function.